beacon-chain/p2p/discovery.go

```diff
@@ -332,6 +332,31 @@ func (s *Service) isPeerAtLimit(inbound bool) bool {
 	return activePeers >= maxPeers || numOfConns >= maxPeers
 }
 
+// PeersFromStringAddrs convers peer raw ENRs into multiaddrs for p2p.
+func PeersFromStringAddrs(addrs []string) ([]ma.Multiaddr, error) {
```

Comment:
Exporting for external use

Reply:
This is the wrong direction imo, we should be having all this under the `network` package as a sub-package for all these helpers.